### PR TITLE
Holding cell for #114

### PR DESCRIPTION
**Pull request by GrumpyOldTroll**
I made one change to the links, because this gets a 404:

    - https://www.scte.org/standards-development/library/standards-catalog/ansiscte-214-1-2016/

  But a web search for "SCTE-214-1" found this instead, which seems to be a new location for the same thing:

    - https://www.scte.org/standards/library/catalog/scte-214-1-mpeg-dash-for-ip-based-cable-services-part1-mpd-constraints-and-extensions/

Additionally, There were 3 more links that failed check-links.sh:

 - https://help.datazoom.io/hc/en-us/articles/360046177191
 - https://help.datazoom.io/hc/en-us/articles/360031323311
 - https://iabtechlab.com/audio-video/tech-lab-digital-video-suite/

All of these get something with chrome, but fail with curl.  The datazoom links get a 403 and the iabtechlab gets a 406.  (Changing the UA header to match chrome's made curl succeed with iabtechlab, but not the datazoom links.  Didn't dig deeper yet.)

I haven't put in automation for this, but I figure it's useful to have something at least to do a quick scan.

### CODE REVIEW

File: draft-ietf-mops-streaming-opcons.md [hunk 1]
```diff
@@ -855,179 +855,11 @@ Unfortunately, as noted in {{RFC7258}}, there is no way to prevent pervasive mon
...
-- OBS Studio: https://obsproject.com/
-- Shaka Player: https://github.com/google/shaka-player
-- Shaka Packager: https://github.com/google/shaka-packager
-- Traffic Control CDN: https://trafficcontrol.apache.org/
-- VideoLAN: https://www.videolan.org/projects/
-- video.js: https://github.com/videojs/video.js
-
-## Technical Events
-
-- ACM Mile High Video (MHV): https://mile-high.video/
```
Comment (acbegen) [inline draft-ietf-mops-streaming-opcons.md:960]: ACM is using the name "ACM Mile-High Video (MHV)" , so there is a dash between mile and high.

File: living-doc-mops-streaming-opcons.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+- OBS Studio: <https://obsproject.com/>
+- Shaka Player: <https://github.com/google/shaka-player>
+- Shaka Packager: <https://github.com/google/shaka-packager>
+- Traffic Control CDN: <https://trafficcontrol.apache.org/>
+- VideoLAN: <https://www.videolan.org/projects/>
+- video.js: <https://github.com/videojs/video.js>
+
+# Technical Events
+
+- ACM Mile High Video (MHV): <https://mile-high.video/>
```
Comment (acbegen) [inline living-doc-mops-streaming-opcons.md:139]: ACM is using the name "ACM Mile-High Video (MHV)" , so there is a dash between mile and high.

**Review by acbegen — APPROVED** [review]
Just make the change for ACM Mile-High Video Conference.

**Comment by acbegen** [comment]
> - https://www.scte.org/standards/library/catalog/scte-214-1-mpeg-dash-for-ip-based-cable-services-part1-mpd-constraints-and-extensions/

This works.

> Additionally, There were 3 more links that failed check-links.sh:
> 
> * https://help.datazoom.io/hc/en-us/articles/360046177191
> * https://help.datazoom.io/hc/en-us/articles/360031323311
> * https://iabtechlab.com/audio-video/tech-lab-digital-video-suite/

These work just fine for me on chrome no 403 or 406, plain 200.

**Comment by GrumpyOldTroll** [comment]
> These work just fine for me on chrome no 403 or 406, plain 200.

They work for me in chrome also, but they don't work in curl or wget so the script complains about them.

**Comment by SpencerDawkins** [comment]
Looks good to me, modulo a Github Action error that we don't understand. Will merge now, so we can move forward.

**Pull request merged**